### PR TITLE
Make it clear that bulk API actions are processed sequentially on each shard rather than sequentially overall

### PR DESCRIPTION
**Pull request by colings86**

### CODE REVIEW

**Review by ywelsch — APPROVED** [review]
LGTM

File: docs/reference/getting-started.asciidoc [hunk 1]
```diff
@@ -551,7 +551,7 @@ POST /customer/external/_bulk?pretty
 
 Note above that for the delete action, there is no corresponding source document after it since deletes only require the ID of the document to be deleted.
 
-The bulk API executes all the actions sequentially and in order. If a single action fails for whatever reason, it will continue to process the remainder of the actions after it. When the bulk API returns, it will provide a status for each action (in the same order it was sent in) so that you can check if a specific action failed or not.
+The bulk API executes all the actions for a given shard sequentially and in order. If a single action fails for whatever reason, it will continue to process the remainder of the actions after it. When the bulk API returns, it will provide a status for each action (in the same order it was sent in) so that you can check if a specific action failed or not.
```
Comment (bleskes) [inline docs/reference/getting-started.asciidoc]: I wonder if we should avoid this all together and just something like "The Bulk API does not fail due to failures of in one of the actions". I think it's simpler?

Comment (colings86) [inline docs/reference/getting-started.asciidoc]: Sure we can change it to that. I assume that this sequential processing per shard is more a consequence of the implementation than a feature of the Bulk API?

Comment (bleskes) [inline docs/reference/getting-started.asciidoc]: > more a consequence of the implementation than a feature of the Bulk API

+1

**Pull request merged**